Rm sourcing of useprod

ingkebil opened the pull request:
This PR removes the soucing of bashrc and the activating of useprod right within the script. Instead, the scripts should use the environment sourced in the current shell.

For testing, see: https://github.com/Clinical-Genomics/servers/pull/262

ingkebil committed:
Rm sourcing of useprod

add log function

ingkebil commented:
Ready for review

patrikgrenfeldt reviewed (commented):
Amazing if this change does what the previous did but better. I don't have the knowledge to judge here, could you explain how this solves the requirements?

ingkebil commented:
I moved `seqtk` alias to a function. Aliases are not transferred between nested bash sessions unless you set `shopt -o expand_aliases` before you set the alias. This is what I removed now. Functions do get transferred from a bash session to the next.
